src/applications/login/containers/SignInApp.jsx

```diff
@@ -52,7 +52,7 @@ export function UnifiedSigninPage({ router, location }) {
   );
 
   const css = `
-  #legacy-header > div:nth-child(3) > div.menu-rule.usa-one-whole,   div .profile-nav-container, .menu-rule.usa-one-whole,  .hidden-header {
```

Comment:
Updated this ID because the desktop header is not "legacy" and this was confusing.